deploy/olm-catalog/performance-addon-operator/4.6.0/performance-addon-operator.v4.6.0.clusterserviceversion.yaml

```diff
@@ -118,6 +118,13 @@ spec:
               labels:
                 name: performance-operator
             spec:
+              affinity:
```

Comment:
it seems OCP control plane is using
```yaml
    nodeSelector:
      node-role.kubernetes.io/master: ""
```
but the end result is the same and affinity rules are more powerful than nodeSelector, so it seems is fine.

Reply:
I preferred affinity since is is reflected in the official docs

Reply:
Is affinity reflected in the openshift official docs or the kubernetes official docs? or both?

Reply:
both